templates/strings/flat-swift4.stencil

```diff
@@ -39,11 +38,13 @@ import Foundation
 {% set enumName %}{{param.enumName|default:"L10n"}}{% endset %}
 {{accessModifier}} enum {{enumName}} {
   {% if tables.count > 1 %}
+  // swiftlint:disable function_parameter_count
```

Comment:
Maybe I misunderstood what @djbe meant by suggesting you to move those next to the enum, but wouldn't it be more consistent to put that rule with the others just 4 lignes above? i.e. add then to the list of already 3 disabled rules there?

```
// swiftlint:disable identifier_name line_length type_body_length function_parameter_count
```
_(of course same for when you re-enable the rule, I'd personally re-enable it in line 52 with the other rules)

I'm probably just nitpicking, but I personally feel easier to maintain having all swiftlint disabled rules grouped next to them when possible/appropriate :wink:

Reply:
Uh yeah, that's what I meant, apologies if that was unclear (I only quickly glanced at the line numbers).

Reply:
I totally agree, I thought about that too,
But I strictly followed the demand 😅